static/sass/styles.scss

```diff
@@ -490,4 +490,37 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 /// can be removed once added upstream
 .is-paper {
   background-color: #f3f3f3;
-}
\ No newline at end of file
+}
+
+.p-department-cards__container {
+  aspect-ratio: 2 / 3;
+  background-color: #fff;
+  margin-bottom: $spv--large;
+  margin-top: $spv--small;
+
+  .p-department-cards__card {
+    @extend .u-stack;
+
+    padding: 0 $sph--large;
+
+    span {
```

Comment:
Why is the colour set on span and not card itself?
Why do we need to set the colour at all?
Also, #000 is not a default text colour in Vanilla, please use $colors--light-theme--text-default or $color-dark.

Reply:
Sorry. Updated, this should be `$color-link` just like in the design. UX asked to make the entire card clickable, so that means the link styles don't get applied to the text.